ibis/tests/expr/test_table.py

```diff
@@ -242,14 +249,16 @@ def test_projection_no_expr(table, empty):
         table.select(empty)
 
 
-def test_projection_invalid_nested_list(table):
-    errmsg = "must be coerceable to expressions"
-    with pytest.raises(com.IbisTypeError, match=errmsg):
-        table.select(["a", ["b"]])
-    with pytest.raises(com.IbisTypeError, match=errmsg):
-        table[["a", ["b"]]]
-    with pytest.raises(com.IbisTypeError, match=errmsg):
-        table["a", ["b"]]
+# FIXME(kszucs): currently bind() flattens the list of expressions, so arbitrary
```

Comment:
Does this need to be fixed?

Reply:
The API is less restrictive now, if we are okay with that then no, otherwise yes.

Reply:
Seems fine to me!

Reply:
Created a follow-up issue for this https://github.com/ibis-project/ibis/issues/7819